### PR TITLE
NES-2140: add command to list and install plugin to resource

### PR DESCRIPTION
**Pull request by ja0x**

### CODE REVIEW

File: nestbox_confd_client/commands/plugins.py [hunk 1]
```diff
@@ -32,8 +32,68 @@ def list(self, tenant_uuid=None, **kwargs):
         self.raise_from_response(r)
         return r.json()
 
+    def list_customer_installs(self, plugin_uuid, tenant_uuid=None, **kwargs):
```
Comment (sduthil) [inline nestbox_confd_client/commands/plugins.py:35]: Documentation is missing in the README

File: nestbox_confd_client/commands/plugins.py [hunk 2]
```diff
@@ -32,8 +32,68 @@ def list(self, tenant_uuid=None, **kwargs):
...
         headers = self._get_headers(tenant_uuid=tenant_uuid)
         url = f'{self.base_url}/{plugin_uuid}'
         r = self.session.put(url, json=plugin, headers=headers)
         self.raise_from_response(r)
+
+    def _list_installs(self, resource_type, plugin_uuid, tenant_uuid=None, **kwargs):
+        headers = self._get_headers(tenant_uuid=tenant_uuid)
+        url = f'{self.base_url}/{plugin_uuid}/installs/{resource_type}'
+        params = {}
+        for key, value in kwargs.items():
```
Comment (sduthil) [inline nestbox_confd_client/commands/plugins.py:84]: Do we really need the loop? E.g.
```
params = dict(kwargs)
if uuids := params.get('uuids'):
    params['uuids'] = ','.join(uuids)
```

**Review by sduthil — CHANGES_REQUESTED** [review]

File: nestbox_confd_client/commands/plugins.py (hunk 2 above)
Comment (ja0x) [reply nestbox_confd_client/commands/plugins.py:84]: +1

File: nestbox_confd_client/commands/plugins.py (hunk 1 above)
Comment (ja0x) [reply nestbox_confd_client/commands/plugins.py:35]: Oops, will add

**Comment by ja0x** [comment]
Unsure about the wording of `install_for_{customer|reseller|location}`.  Should it be `install_to_{customer|reseller|location}` to make it clearer?

**Review by sduthil — APPROVED** [review]

**Pull request merged**